control/pid_controller_dp/include/pid_controller_dp/pid_controller.hpp

```diff
@@ -0,0 +1,47 @@
+#ifndef PID_CONTROLLER_HPP
+#define PID_CONTROLLER_HPP
+
+#include "pid_controller_dp/eigen_typedefs.hpp"
+
+class PIDController {
+   public:
+    explicit PIDController();
+
+    // @brief Calculate the control input tau
+    // @param eta: 7D vector containing the vehicle pose [x, y, z, w, x, y, z]
+    // @param eta_d: 7D vector containing the desired vehicle pose [x, y, z, w,
+    // x, y, z]
+    // @param nu: 6D vector containing the vehicle velocity [u, v, w, p, q, r]
+    // @param eta_dot_d: 7D vector containing the desired vehicle velocity [u,
+    // v, w, p, q, r]
+    // @return 6D vector containing the control input tau [u, v, w, p, q, r]
```

Comment:
That is not how tau looks.

Reply:
[Fx, Fy, Fz, torque_x, torque_y, torque_z], Dette bedre?

Reply:
[X,Y,Z, K, M, N]